prime-router/src/main/kotlin/history/azure/HistoryDatabaseAccess.kt

```diff
@@ -229,11 +229,13 @@ abstract class HistoryDatabaseAccess(
     ): Condition {
         var filter = this.organizationFilter(organization, orgService)
 
-        var statusList = receivingOrgSvcStatus?.map { it.name.lowercase() }
-        filter = filter.and(
+        if (receivingOrgSvcStatus != null) {
```

Comment:
Are there tests that verify this new logic?

Reply:
There are existing tests where if you do not pass this, the response status is validated.